.editorconfig

```diff
@@ -0,0 +1,17 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+trim_trailing_whitespace = true
+end_of_line = lf
+insert_final_newline = true
+
+# Tab indentation (no size specified)
+[Makefile]
+indent_style = tab
+
+# cgo code
+[*.go.h]
+indent_size = 2
```

Comment:
From the code it doesn't look like this actually gave the correct result. 

However, I talked about this with @oleorhagen, and we decided that maybe it's better to just standardize the C code template to something we use in the rest of the company, instead of something that matches Golang. So my suggestion is, use [this one](https://github.com/mendersoftware/mender-binary-delta/blob/master/.clang-format) instead. It's originally taken from CFEngine, and used in mender-binary-delta.

Together with that, you should add [a clang install](https://github.com/mendersoftware/mender-binary-delta/blob/master/.gitlab-ci.yml#L85) and [a check](https://github.com/mendersoftware/mender-binary-delta/blob/master/.gitlab-ci.yml#L105).

Does that sound good?

Reply:
Let me try :)

Reply:
Done!

Reply:
This looks better, thanks!

Reply:
I also added the check in the pipeline, let's see if it passes.